pandas/core/generic.py

```diff
@@ -10397,43 +10397,50 @@ def _doc_parms(cls):
 Series([], dtype: bool)
 """
 
-_sum_examples = """\
+_shared_docs['stat_func_example'] = """\
 Examples
 --------
-``MultiIndex`` series example of monthly rainfall
+``MultiIndex`` series example.
 
->>> index = pd.MultiIndex.from_product(
-...     [['London', 'New York'], ['Jun', 'Jul', 'Aug']],
-...     names=['city', 'month'])
->>> s = pd.Series([47, 35, 54, 112, 117, 113], index=index)
+>>> index = pd.MultiIndex.from_arrays([
+...     ['Warm', 'Warm', 'Cold', 'Cold'],
+...     ['Dog', 'Falcon', 'Fish', 'Spider']],
+...     names=['Blooded', 'Animal'])
+>>> s = pd.Series([4, 2, 0, 8], name='Legs', index=index)
```

Comment:
I think in the rest of the documentation we use the values, columns... (e.g. dog, warm, blooded...) in lower case. Can you take a quick look to confirm, and change it please.

And we use `idx` for the index.

Reply:
Done